### PR TITLE
Add functions to fetch funding rate data from market

### PR DESCRIPTION
**Pull request by Issengaard**
The market client has been modified to enable the retrieval of funding rate data with the following changes:
- `NewFundingRateHistoryService` now fetches funding rate history data using the Rest API.
- `NewFundingRateInfoService` now retrieves symbols' funding rate information using the Rest API.
Tests for the new functions have been created following the same approach as other tests in the package.

### CODE REVIEW

File: README.md
Comment (ldt116) [inline README.md:1]: This Readme change should not be included in the PR

**Review by mm2175 — CHANGES_REQUESTED** [review]
Please do not change import path.

**Comment by Issengaard** [comment]
Good day, I close this request to open new with correct data.
